pkg/conn/baseconn.go

```diff
@@ -160,9 +160,11 @@ func (conn *BaseConn) ExecuteSQLWithIgnoreError(tctx *tcontext.Context, hVec *me
 			arg = args[i]
 		}
 
-		tctx.L().Debug("execute statement",
-			zap.String("query", utils.TruncateString(query, -1)),
-			zap.String("argument", utils.TruncateInterface(arg, -1)))
+		if tctx.L().Check(zap.DebugLevel, "test") != nil {
```

Comment:
how does this change take effect?

Reply:
If not debug level, this func will return nil, and will not call TruncateInterface

Reply:
@GMHDBJD you may add a comment here to explain it (for the future).